{
	"id": "AVG-1288",
	"advisories": [
		{
			"content": {
				"id": "ASA-202011-25",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/ASA-202011-25"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "AVG-1288",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "High"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-1288"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-28896",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2020-28896"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "arch",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"vulnerable": true,
								"fix_status": {
									"class": "fixed"
								},
								"package": {
									"name": "mutt"
								},
								"affected": {
									"type": "pacman",
									"range": [
										{
											"lt": "2.0.2-1"
										}
									],
									"fixed": [
										"2.0.2-1"
									]
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-1288.json"
		]
	}
}
